src/index.ts

```diff
@@ -53,6 +67,43 @@ export const beautifier: Beautifier = {
   },
 };
 
+function findFile({
+  finishPath = "/",
+  startPath = finishPath,
+  fileNames,
+}: {
+  startPath: string | undefined;
+  finishPath: string | undefined;
+  fileNames: string[];
+}): Promise<string> {
+  // Recursive case: Check & go uuuuppp
+  const filePaths = fileNames.map(fileName => path.join(startPath, fileName));
+  // tslint:disable-next-line
```

Comment:
What is this disabling?

Reply:
`No need to wrap 'doesFileExist' in another function. Just use it directly.` . https://palantir.github.io/tslint/rules/no-unnecessary-callback-wrapper/